kotsadm/pkg/automation/automation.go

```diff
@@ -86,10 +89,20 @@ func AutomateInstall() error {
 			continue
 		}
 
+		skipImagePush := false
+		kotsadmConfigMap, err := clientset.CoreV1().ConfigMaps(os.Getenv("POD_NAMESPACE")).Get(context.TODO(), kotsadmtypes.KotsadmConfigMap, metav1.GetOptions{})
+		if err != nil {
+			if !kuberneteserrors.IsNotFound(err) {
+				return errors.Wrap(err, "failed to get existing kotsadm config map")
+			}
+		} else if err == nil {
+			skipImagePush, _ = strconv.ParseBool(kotsadmConfigMap.Data["initial-app-images-pushed"])
```

Comment:
should this whole 'get skipImagePush' section be a function? since it's also used here: https://github.com/replicatedhq/kots/pull/1076/files#diff-963b65faecc60bcb8286f696a69cf87eR282